<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingToolWid</class>
 <widget class="QWidget" name="SettingToolWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="timeBtn">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>485</y>
     <width>93</width>
     <height>33</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdBtn">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>335</y>
     <width>93</width>
     <height>33</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>15</y>
     <width>72</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>级联数量</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>15</y>
     <width>49</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>33</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>436</y>
     <width>72</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>时间间隔</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="outputSpinBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>198</y>
     <width>49</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>33</number>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="outputNumBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>93</width>
     <height>33</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="devNumBtn">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>65</y>
     <width>93</width>
     <height>33</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QComboBox" name="timeBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>436</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>500ms</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1000ms</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1500ms</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>290</y>
     <width>72</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>命令模式</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>198</y>
     <width>54</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>输出位</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="cmdBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>单命令</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>双命令</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>相数</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="lineSpinBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>3</number>
   </property>
  </widget>
  <widget class="QPushButton" name="lineBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
